aissemble-universal-config-loader/pyproject.toml

```
@@ -0,0 +1,39 @@
+[project]
+name = "aissemble-universal-config-loader"
+version = "1.0.0.dev"
+description = "aiSSEMBLE Universal Config Loader"
+authors = [{name = "Chaoyi Sun", email = "csun@cpointe-inc.com"}]
+
+readme = "README.md"
+requires-python = ">=3.11.4"
+license = { text = "Booz Allen Public License v1.0" }
+license-files = ["LICENSE*"]
+dynamic = ["version", "dependencies"]
+dependencies = [
+    "krausening (>=20,<21)",
```

Same here, would expect
```suggestion
    "krausening>=20",
```